src/sentry/models/event.py

```diff
@@ -496,7 +493,9 @@ def datetime(self):
     def message(self):
         if "message" in self.snuba_data:
             return self.snuba_data["message"]
-        return self.data.get("message")
+        from sentry.event_manager import EventManager
+        event_manager = EventManager(self.data.data)
```

Comment:
Isn't EventManager depending on Event itself ? Could we avoid this dependency by moving get_search_message ? search_message feels like a property of an event.